react.js

```diff
@@ -0,0 +1,55 @@
+"use strict";
+
+const baseConfig = require(".");
+
+const eslintReactRecommendedRules = {
+  "react/display-name": 2,
+  "react/jsx-key": 2,
+  "react/jsx-no-comment-textnodes": 2,
+  "react/jsx-no-duplicate-props": 2,
+  "react/jsx-no-target-blank": 2,
+  "react/jsx-no-undef": 2,
+  "react/jsx-uses-react": 2,
+  "react/jsx-uses-vars": 2,
+  "react/no-children-prop": 2,
+  "react/no-danger-with-children": 2,
+  "react/no-deprecated": 2,
+  "react/no-direct-mutation-state": 2,
+  "react/no-find-dom-node": 2,
+  "react/no-is-mounted": 2,
+  "react/no-render-return-value": 2,
+  "react/no-string-refs": 2,
+  "react/no-unescaped-entities": 2,
+  "react/no-unknown-property": 2,
+  "react/no-unsafe": 0,
+  "react/react-in-jsx-scope": 2,
```

Comment:
hur lirar detta med react i runtime? (ej förkompilerad)
då är denna regel ej längre relevant

Reply:
vilken av reglerna, no-unescaped? varför är den inte relevant?

Reply:
`react-in-jsx-scope`? att källan till `createElement` alltid ska importerad?
vi hade det i maisha när vi körde jsx i require hook. var inget problem då. nu bygger vi vyerna i Docker-filer inget problem då heller.

Reply:
inte nödvändigtvis `createElement` utan detta:
```
import React from "react";

export function MyComponent() {
  return <div/>
}
```
I ovan exempel används `React` aldrig nånstans, men måste finnas i scope om det är jsx. Behövs inte med `jsx-runtime` dock

Reply:
när det blir aktuellt med jsx som runtime så får vi lägga till en motsvarande jsx-runtime (PR tas gärna om sådant case redan finnes!)